content/gsoc/2025/_index.md

```diff
@@ -52,11 +54,54 @@ The type system design will require particular attention to several key areas. T
 
 ### Potential mentors
 
-
 * [Saransh Chopra](https://Saransh-cpp.github.io/)
 * [Agriya Khetarpal](https://github.com/agriyakhetarpal/)
 
 <!-- * [Valentin Sulzer](https://github.com/valentinsulzer)
 * [Robert Timms](https://github.com/rtimms)
 * [Arjun Verma](https://arjxnpy.vercel.app/)
 * [Ferran Brosa Planella](https://www.brosaplanella.xyz/) -->
+
+## pytest-ifying the testing suite
+
+PyBaMM (Python Battery Mathematical Modelling) used `unittest` as its testing framework until last year. However, thanks to the efforts from [@prady0t](https://github.com/prady0t), it now uses `pytest.` `pytest` is a modern and community-maintained testing framework for Python that is more flexible and feature-rich than the built-in `uni test` module. Migrating such a huge codebase from `unittest` to `pytest` was an exceptional feat; however, because of limited time and resources, we could not leverage several amazing things that `pytest` offers. There have been efforts to introduce `pytest` specific features, but these efforts have been distributed, and now the codebase is at an awkward stage where some tests utilize `pytest`'s capabilities and some do not.
```

Comment:
```suggestion
PyBaMM (Python Battery Mathematical Modelling) used `unittest` as its testing framework until last year. However, thanks to the efforts from [@prady0t](https://github.com/prady0t), it now uses `pytest.` `pytest` is a modern and community-maintained testing framework for Python that is more flexible and feature-rich than the built-in `unittest` module. Migrating such a huge codebase from `unittest` to `pytest` was an exceptional feat; however, because of limited time and resources, we could not leverage several amazing things that `pytest` offers. There have been efforts to introduce `pytest` specific features, but these efforts have been distributed, and now the codebase is at an awkward stage where some tests utilize `pytest`'s capabilities and some do not.
```